src/H5Oefl.c

```diff
@@ -90,36 +85,38 @@ H5O__efl_decode(H5F_t *f, H5O_t H5_ATTR_UNUSED *open_oh, unsigned H5_ATTR_UNUSED
     /* Check args */
     HDassert(f);
     HDassert(p);
-    HDassert(p_size > 0);
```

Comment:
So, does p_size need to be checked?  I haven't seen it being checked/asserted in other functions

Reply:
No. If it's 0 the first check will fail and we'll jump to the error condition.